app/grandchallenge/archives/views.py

```diff
@@ -558,6 +564,128 @@ def get_serializer_class(self):
             return ArchiveItemSerializer
 
 
+class ArchiveItemJobListView(
+    LoginRequiredMixin,
+    ObjectPermissionRequiredMixin,
+    PaginatedTableListView,
+):
+    model = Job
+    template_name = "archives/archive_item_job_list.html"
+    row_template = "archives/archive_item_job_list_row.html"
+
+    search_fields = [
+        "pk",
+        "creator__username",
+        "comment",
+        # Inputs
+        "inputs__image__name",
+        "inputs__image__files__file",
+        # Outputs
+        "outputs__interface__title",
+        "outputs__interface__description",
+        # Algorithm
+        "algorithm_image__algorithm__title",
+        "algorithm_image__algorithm__description",
+        "algorithm_image__comment",
+    ]
```

Comment:
Having so many search fields with so many joins will make the search impossible to use.

Reply:
Good to know. I wasn't too sure about the practical effects. I do think we should:
- to open up the search for diversity in outputs 
- make it easy to find specific algorithms

How about:
```Python
search_fields = [
        "pk",
        # Outputs
        "outputs__interface__title",
        # Algorithm
        "algorithm_image__algorithm__title",
        "algorithm_image__algorithm__description",
        "algorithm_image__comment",
    ]
```

The `algorithm_image__comment` one would allow to search for a specific version of an algorithm.

Reply:
Joins and free text are expensive to search, I would get rid of both the description and the comment.